src/UpsilonGC/UpsilonGCHeap.cpp

```diff
@@ -500,15 +503,15 @@ bool UpsilonGCHeap::IsInFrozenSegment(Object* object)
 Object * UpsilonGCHeap::AllocLHeap(size_t size, uint32_t flags)
 {
     int sizeWithHeader = size + sizeof(ObjHeader);
-    ObjHeader* address = (ObjHeader*)calloc(sizeWithHeader, sizeof(char*));
+    ObjHeader* address = (ObjHeader*)malloc(sizeWithHeader * sizeof(char*));
```

Comment:
This breaks the allocator contract - we need zero initialized memory returned by `AllocLHeap`.